### PR TITLE
#3761 move from node-sass to sass in presentation server

### PR DESCRIPTION
**Pull request by davidradl**
Signed-off-by: David Radley <david_radley@uk.ibm.com>

I brought up the Ui -it all looks the same

### CODE REVIEW

**Review by planetf1 — APPROVED** [review]
Good news , I do think this will simplify things

File: open-metadata-implementation/user-interfaces/presentation-server/nodejs/package-lock.json [hunk 1]
```diff
@@ -5197,20 +5044,100 @@
...
       "dev": true,
-      "optional": true,
       "requires": {
-        "bindings": "^1.5.0",
-        "nan": "^2.12.1",
-        "node-pre-gyp": "*"
+        "global-prefix": "^3.0.0"
       },
       "dependencies": {
+<<<<<<< HEAD
```
Comment (grahamwallis) [inline open-metadata-implementation/user-interfaces/presentation-server/nodejs/package-lock.json:5135]: Need to clean up the residue from Git conflict resolutions

Comment (grahamwallis) [reply open-metadata-implementation/user-interfaces/presentation-server/nodejs/package-lock.json:5135]: There are multiple occurrences in this file.

Comment (davidradl) [reply open-metadata-implementation/user-interfaces/presentation-server/nodejs/package-lock.json:5135]: Good spot @grahamwallis I missed that

**Pull request merged**